ansible/roles/openshift_logging/tasks/main.yml

```diff
@@ -166,6 +166,16 @@
   when: podout.results.results[0]['items'][0].status.phase != 'Succeeded'
   run_once: true
 
+- name: add cluster role binding to aggregated-logging-elasticsearch service account
+  oadm_policy_user:
```

Comment:
My only concern here is that the `oadm_policy_user` module is now out of date and we should be deprecating them in favor of the upstream code in the future.  That should happen once we arrive at 3.5.

That being said this should work.